src/types/schema.ts

```diff
@@ -1,65 +1,108 @@
 import { UUID } from 'crypto';
 
-export interface CaseListing {
-  id: UUID;
-  title: string;
-  summary: string;
-  country: string;
-  client_location: string;
-  legal_server_id: number;
-  hours_per_month: number;
-  num_months: number;
-  languages: string[];
-  is_remote: boolean;
-  in_court: boolean;
-  needs_attorney: boolean;
-  needs_interpreter: boolean;
-  upcoming_date: string;
-  relief_sought: string;
-  active: boolean;
+// table field types
+export type ImmigrationLawExperienceEnum =
+  | 'No Experience'
+  | 'One Experience'
+  | 'Multiple Experiences';
+
+export type RoleEnum =
+  | 'Attorney'
+  | 'Interpreter'
+  | 'Research Fellow'
+  | 'Translator';
+
+export type ListingTypeEnum =
+  | 'Case'
+  | 'Limited Assistance'
+  | 'Translation Request';
+
+// join table rows
+export interface UserLanguage {
```

Comment:
Curious to hear your thought process on naming this `UserLanguage` as opposed to `ProfileLanguage`? I think if we do `UserLanguage` here I'm almost inclined to rename `Profile` to `UserProfile`....

Reply:
Main reason is that if I named it `ProfileLanguage`, then I'll have to name the other one `ProfileRole`, which sounds a lot like `ProfileRow` and could get confusing... Thoughts?